thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdEyeAnomalyConfiguration.java

```diff
@@ -4,8 +4,11 @@
 import com.linkedin.thirdeye.anomaly.task.TaskDriverConfiguration;
 import com.linkedin.thirdeye.auto.onboard.AutoOnboardConfiguration;
 import com.linkedin.thirdeye.common.ThirdEyeConfiguration;
+import java.util.List;
+
 
 public class ThirdEyeAnomalyConfiguration extends ThirdEyeConfiguration {
+  private boolean holidayEventsLoader = false;
```

Comment:
Since you are modifying this part of code, could you order the flags alphabetically? And remove the deprecated isMerger flag. Thanks.

Reply:
Sure!